config/config.yaml

```diff
@@ -3,3 +3,27 @@
 # one row per sample. It can be parsed easily via pandas.
 samples: "config/samples.tsv"
 units: "config/units.tsv"
+
+params:
+  kallisto: "-b 100"
+  # these cutadapt parameters need to contain the required flag(s) for
+  # the type of adapter(s) to trim, i.e.:
+  # * https://cutadapt.readthedocs.io/en/stable/guide.html#adapter-types
+  #   * `-a` for 3' adapter in the forward reads
+  #   * `-g` for 5' adapter in the forward reads
+  #   * `-b` for adapters anywhere in the forward reads
+  # also, separate capitalised letter flags are required for adapters in
+  # the reverse reads of paired end sequencing:
+  # * https://cutadapt.readthedocs.io/en/stable/guide.html#trimming-paired-end-reads
+  cutadapt-se: ""
```

Comment:
As a good default, [Illumina TruSeq adapters as specified in this document](https://support.illumina.com/downloads/illumina-adapter-sequences-document-1000000002694.html) are probably a good default, as this is very commonly used sequencing methodology. For `cutadapt-se`, the `Universal Adapter` at the `5'` end probably makes most sense:
```
  cudatapt-se: "-g AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT"
```